app/src/components/Register.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import axios from 'axios'
+
+class Register extends React.Component {
+  state = {
+    username: '',
+    password: ''
+  }
+  inputHandler = e => {
+    let value = e.target.value
+    this.setState({ [e.target.name]: value })
+  }
+  submit = e => {
+    e.preventDefault()
+    const { username, password } = this.state
+    axios
+      .post(`http://localhost:9090/api/register`, {
+        username,
+        password
+      })
+      .then(res1 => {
+        axios
+          .post(`http://localhost:9090/api/login`, {
```

Comment:
Couldn't the register endpoint be modified to simply return a token to do this rather than making two API calls? I think it'd definitely be a little more efficient!